net-libs/libsrsirc/libsrsirc-0.0.14.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Lightweight, cross-platform IRC library"
+HOMEPAGE="https://github.com/fstd/libsrsirc"
+SRC_URI="https://kiwi.pr0.tips/${P}.tar.gz"
```

Comment:
Uh, what is this website?

Reply:
New upstream host:
https://github.com/fstd/libsrsirc/issues/6#issuecomment-2081234589
> I've created a new release, 0.0.14.1 to address this issue; could you check whether it works? Also note the tarball is now hosted at https://kiwi.pr0.tips/libsrsirc-0.0.14.1.tar.gz instead of at the former domain (penenen.de).

Reply:
Will note in commit later.